content/influxdb/cloud/write-data/troubleshoot.md

```diff
@@ -18,45 +18,48 @@ related:
-For example, a partial write may occur when InfluxDB writes all points that conform to the bucket schema, but rejects points that have the wrong data type in a field.
+For example, a partial write may occur when InfluxDB writes points that conform to the schema of existing data, but rejects points that have the wrong data type in a field.
 
-## HTTP status codes
+## Review HTTP status codes
 
 InfluxDB uses conventional HTTP status codes to indicate the success or failure of a request.
 Write requests return the following status codes:
 
 - `204` **Success**: InfluxDB validated the request data format and accepted the data for writing to the bucket.
     {{% note %}}
-  `204` doesn't indicate a successful write operation since writes are asynchronous.
-  If some of your data did not write to the bucket, see how to [troubleshoot rejected points](#troubleshoot-rejected-points).
+`204` doesn't indicate a successful write operation since writes are asynchronous. If some of your data did not write to the bucket, see how to [check for rejected points](#review-rejected-points).
     {{% /note %}}
 
-- `400` **Bad request**: The [line protocol](/influxdb/cloud/reference/syntax/line-protocol/) data in the request was malformed.
-   The response body contains the first malformed line in the data. All request data was rejected and not written.
+- `400` **Bad request**: The [line protocol](/influxdb/cloud/reference/syntax/line-protocol/) data in the request is malformed.
+   The response body contains the first malformed line in the data. All request data is rejected and not written.
 - `401` **Unauthorized**: May indicate one of the following:
   -  [`Authorization: Token` header](/influxdb/cloud/api-guide/api_intro/#authentication) is missing or malformed.
   - [API token](/influxdb/cloud/api-guide/api_intro/#authentication) value is missing from the header.
-  - API token does not have sufficient permissions to write to the organization and bucket. For more information about token types and permissions, see [Manage API tokens](/influxdb/cloud/security/tokens/)
-- `404` **Not found**: A requested resource (e.g. an organization or bucket) was not found. The response body contains the requested resource type, e.g. "organization", and resource name.
-- `413` **Request entity too large**: The payload exceeded the 50MB limit. All request data was rejected and not written.
+  - API token does not have sufficient permissions to write to the organization and bucket.
+    For more information about token types and permissions, see [Manage API tokens](/influxdb/cloud/security/tokens/)
+- `404` **Not found**: A requested resource (e.g. an organization or bucket) was not found.
+  The response body contains the requested resource type, e.g. "organization", and resource name.
+- `413` **Request entity too large**: The request payload exceeded the size limit (50 MB uncompressed data).
```

Comment:
```suggestion
- `413` **Request entity too large**: The write request payload exceeded the size limit (**50 MB *uncompressed*** data or **250 MB *decompressed***).
```

Reply:
InfluxDB supports 50MB of compressed data and 250MB of decompressed data per write request payload. The request exceeds these limits, resulting in a 413 `Request too large` error.  (TBD: do we distinguish exceeding compressed or decompressed issue in message?)